apps/www/next.config.js

```diff
@@ -34,11 +34,6 @@ module.exports = withTM(
     async rewrites() {
       return {
         beforeFiles: [
-          // /front is only accessible via _middleware rewrite
-          {
-            source: '/front',
-            destination: '/404',
-          },
```

Comment:
why remove this?

Reply:
It no longer works in combination with the middleware. 
If we rewrite `/` -> `/front`, with Next.js 12.2 `/` is rewritten to `/404`. 
So in order to fix that I moved rewriting `/front` -> `/404` into the middleware, which doesn't clash with the first mentioned rewrite (which is also done in the middleware)

Reply:
makes sense, didn't notice it in the middleware.